app/forms/idv/finance_form.rb

```diff
@@ -10,25 +11,18 @@ class FinanceForm
 
     attr_reader :idv_params, :finance_type, *FINANCE_TYPES
 
-    validates :finance_type, presence: true
-
     FINANCE_TYPES.each do |type|
       validates type, presence: true, if: ->(form) { form.finance_type == type }
```

Comment:
I couldn't figure out how to get this into the validator..I think I might need activemodel?